Guide-for-using-MongoDB-and-deploying-to-Heroku.md

```diff
@@ -0,0 +1,114 @@
+##Guide for using MongoDB and deploying to Heroku
+
+In this guide lets see how to work with MongoDB locally and with mLab for deploying it to Heroku. Alternatively you can also use mLab add-on in Heroku , It is free but it may require your Credit card Details. So, if you are not intrested in providing your credit card details you can go with [mLab](https://mlab.com) website. 
+
+#### Setting up a free account on Heroku and mLab:
+
+Sign up for [Heroku](https://signup.heroku.com/) and [mLab](https://mlab.com/signup/)
+
+
+####Starting your Mongodb(Locally): 
+
+```mongod --port 27017 --dbpath=./data``` _(Create a directory named 'data' if it doesn't exist)_
+
+Now your Db is running at-
+
+`mongodb://localhost:27017/my_database_name`
+
+>If you are using C9 and If you are having trouble starting your DB in C9 refer to this [page](https://community.c9.io/t/setting-up-mongodb/1717)
```

Comment:
To show something optional?? its my first time i am doing this .. so.. so i should remove the block-Quotations?